.github/workflows/tests.yaml

```diff
@@ -22,10 +22,11 @@ jobs:
             matrix:
                 php: ['8.1']
                 path:
-                    - tests
-                    - rules-tests
-                    - packages-tests
-                    - utils-tests
+                    - "tests"
+                    - "--testsuite php-rules"
+                    - "--testsuite other-rules"
+                    - "packages-tests"
+                    - "utils-tests"
```

Comment:
we need the separate items here in this file, as each list-item is executed as a separate CI job.

if we consolidate it into a single item it will no longer run in parallel

Reply:
sorry for not having been clear, I meant keep a list here, that will represent a list of test suite names:
```
                    - tests
                    - php-rules
                    - other-rules
                    - packages-tests
                    - utils-tests
```
create the missing testsuites and adapt the run command in the job

Reply:
That's why I suggest to rename also the matrix param from `path` to `testsuite` :)